homeassistant/components/wallbox/sensor.py

```diff
@@ -34,10 +33,12 @@ class WallboxSensor(CoordinatorEntity, SensorEntity):
     def __init__(self, coordinator, idx, ent, config):
         """Initialize a Wallbox sensor."""
         super().__init__(coordinator)
-        self._properties = CONF_SENSOR_TYPES[ent]
-        self._name = f"{config.title} {self._properties[CONF_NAME]}"
-        self._icon = self._properties[CONF_ICON]
-        self._unit = self._properties[CONF_UNIT_OF_MEASUREMENT]
+        _properties = CONF_SENSOR_TYPES[ent]
+
+        self._name = f"{config.title} {_properties[CONF_NAME]}"
+        self._icon = _properties[CONF_ICON]
+        self._unit = _properties[CONF_UNIT_OF_MEASUREMENT]
```

Comment:
```suggestion
        self._attr_unit_of_measurement = _properties[CONF_UNIT_OF_MEASUREMENT]
```
and remove the property unit_of_measurement